roles/splunk_common/tasks/wait_for_splunk_instance.yml

```diff
@@ -1,15 +1,12 @@
 ---
 - name: Check Splunk instance is running
   uri:
-    url: "{{ cert_prefix }}://{{ splunk_instance_address }}:{{ splunk.svc_port }}/services/server/info?output_mode=json"
+    url: "{{ cert_prefix }}://{{ splunk_instance_address }}:{{ splunk.svc_port }}"
     method: GET
-    user: "{{ splunk.admin_user }}"
-    password: "{{ splunk.password }}"
     validate_certs: false
   register: task_response
   until:
     - task_response.status == 200
-    - lookup('pipe', 'date +"%s"')|int - task_response.json.entry[0].content.startup_time > 10
```

Comment:
cc @arctan5x do you remember what this check was for? I wonder if it's safe to remove now that most of this provisioning is done asynchronously

Reply:
Splunk used to restart couple times through Ansible provisioning and wanted to give some time after Splunk starts to stabilize.